io.catenax.fleet.diagnostic_data/2.0.0/DiagnosticData.ttl

```diff
@@ -0,0 +1,541 @@
+   samm:preferredName "ECU assembly part number version"@en ;
+   samm:description "OEM-specific ecu assembly version"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "0001" .
+
+:catenaXPartId a samm:Property ;
+   samm:preferredName "Catena-X Identifier of this part"@en ;
+   samm:description "The fully anonymous and unique Catena-X ID of this part. This id is unique inside Catena-X data space."@en ;
+   samm:characteristic ext-uuid:UuidV4Trait ;
+   samm:exampleValue "430f56d3-1234-1234-1234-efab12341234" .
+
+:calibrationVersion a samm:Property ;
+   samm:preferredName "Calibration version"@en ;
+   samm:description "Calibration version that was put on this ecu. A calibration file works like a parameter file for the ecu software."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Calibration_file_4711" .
+
+:hwPartNumber a samm:Property ;
+   samm:preferredName "ECU HW part number"@en ;
+   samm:description "hardware part number of ECU"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "04C907309AE" .
+
+:hwVersion a samm:Property ;
+   samm:preferredName "ECU HW version"@en ;
+   samm:description "hardware version of ECU"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "0556A" .
+
+:oemPartName a samm:Property ;
+   samm:preferredName "Oem part name"@en ;
+   samm:description "Part name as defined by the original equipment manufacturer (OEM). Typically the OEM is installing the part in the car."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Gear box M63" .
+
+:oemPartNumber a samm:Property ;
+   samm:preferredName "OEM part number"@en ;
+   samm:description "Part number of the original equipment manufacturer (OEM). Typically the OEM is installing the part in the car. A part number is NOT unique: All parts of the same kind share the same part number."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "FZ206460050202212" .
+
+:readOutDate a samm:Property ;
+   samm:preferredName "Read out date"@en ;
+   samm:description "Date when this ECU information was read out from the diagnostic session"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-01-30T14:45:54"^^xsd:dateTime .
+
+:swPartNumber a samm:Property ;
+   samm:preferredName "ECU SW part number"@en ;
+   samm:description "SW part number of this ecu"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "04C906026BH" .
+
+:swVersion a samm:Property ;
+   samm:preferredName "ECU SW version"@en ;
+   samm:description "current version of the software on this ecu"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "0001" .
+
+:variantCoding a samm:Property ;
+   samm:preferredName "Variant coding"@en ;
+   samm:description "Variant coding that was put on this ecu. Variant coding tells the ecus software how to behave."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Variant: No trailer." .
+
+:dtcList a samm:Property ;
```

Comment:
Property and the referenced Characteristic should not have the same name

Reply:
fixed with next upload